### DISCUSSION TITLE
How can I change the "--q-transition-duration" of the q-menu of the dropdown button

### DISCUSSION THREAD

**Discussion by ninesunqian** [quasarframework/quasar#15927] [General - Components / Directives / etc] [answered]
Hello,
How can I change the "--q-transition-duration" of the q-menu of the dropdown button ? The default value is 300ms, too long for me. I need 0ms.

Thanks very much.

**Comment by suleimanas** [comment, marked as answer]
Hi, By using `transition-duration` prop on QMenu set the transition duration (in milliseconds, without unit)

```
<q-btn ...>
  <q-menu ... :transition-duration="0">
    ...
  </q-menu>
</q-btn>
```

https://quasar.dev/vue-components/menu#qmenu-api
